src/conv.rs

```diff
@@ -1,19 +1,19 @@
 use crate::{
-    value::{FromValue, ToValue, Value},
-    Tag,
+    value::{FromValue, IntoValue, Value},
+    Runtime, Tag,
 };
 
 macro_rules! value_i {
     ($t:ty) => {
-        unsafe impl ToValue for $t {
-            fn to_value(self) -> $crate::Value {
-                $crate::Value::int(self as crate::Int)
+        unsafe impl IntoValue for $t {
```

Comment:
One mismatch between ocaml-interop and ocaml-rs is:

- `FromOCaml<OCamlT> for RustT` vs `FromValue for RustT`
- `ToOCaml<OCamlT> for RustT` vs `ToValue for RustT`

The idea for the API I went with in ocaml-interop is that one value could support multiple conversions, but maybe it is not necessary to do it this way and it would be simpler to just use intermediary types. This is something I am still not decided on, not sure if you have any thoughts on this.

Reply:
I think having the option to support multiple conversions is nice, but I left this in because I don't see the harm in providing an option (and the 1-to-1 type conversion can be simpler in some cases). But since `ocaml-interop` values can convert to and from `ocaml-rs` values it should be possible to use `OCaml<T>` arguments and return types too.